icekit/contrib/navigation/README.md

```diff
@@ -0,0 +1,29 @@
+icekit.contrib.navigation
+=========================
+
+
+Installation
+------------
+
+In project settings
+
+```
+INSTALLED_APPS += (
+    'icekit.contrib.navigation',
+)
+
+FLUENT_CONTENTS_PLACEHOLDER_CONFIG['navigation_content'] = {
+    'plugins': DEFAULT_NAVIGATION_CONTENT_PLUGINS,
```

Comment:
The setting is `ICEKIT_CONTRIB_NAVIGATION_PLUGINS`, but I'd propose `IK_NAVIGATION_PLUGINS`.